libs/neo4j/pyproject.toml

```diff
@@ -54,6 +54,7 @@ optional = true
 
 [tool.poetry.group.dev.dependencies]
 langchain-core = {git = "https://github.com/langchain-ai/langchain.git", subdirectory = "libs/core"}
+langchain-openai = "^0.2.8"
```

Comment:
Looks like langchain-openai is still here. Might be worth just dropping any changes to the poetry files for this PR unless you've added any other deps